ci-operator/step-registry/ipi/conf/mirror/ipi-conf-mirror-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+mkdir -p ${HOME}/.docker
+cp ${CLUSTER_PROFILE_DIR}/pull-secret ${HOME}/.docker/config.json
+oc registry login
+MIRROR_BASE=$( oc get is release -o 'jsonpath={.status.publicDockerImageRepository}' )
+oc adm release new \
+  --from-release ${RELEASE_IMAGE_LATEST} \
+  --to-image ${MIRROR_BASE}-scratch:release \
+  --mirror ${MIRROR_BASE}-scratch \
+  || echo 'ignore: the release could not be reproduced from its inputs'
+oc adm release mirror \
+  --from ${MIRROR_BASE}-scratch:release \
+  --to ${MIRROR_BASE} \
+  --to-release-image ${MIRROR_BASE}:mirrored
+oc delete imagestream "$(basename "${MIRROR_BASE}-scratch")"
```

Comment:
Also important to the mirroring flow is clobbering the provided install target with the scratch release's pullspec.  Clayton pointed out that that retarget is important to ensure that the cluster is indeed consuming the mirrored release, because the original `RELEASE_IMAGE_LATEST` is still available.  I was thinking some well-known filename in `SHARED_DIR` or some such as an ipi-install API for "please ignore `RELEASE_IMAGE_LATEST` and use `$PULLSPEC` instead".